modules/core/src/mathfuncs_core.simd.hpp

```diff
@@ -31,6 +31,11 @@ using namespace cv;
 
 namespace {
 
+static const float atan2_p1 = 0.9997878412794807f*(float)(180/CV_PI);
```

Comment:
Looks like changes in this file are not necessary.

Reply:
These varibles are also used in some place like https://github.com/opencv/opencv/blob/1ae001d10a9b1245c73ccf93fb5d612443108c0a/modules/core/src/mathfuncs_core.simd.hpp#L82

So I think it make sense to modify the code here, right?